drivers/spi/spi_mcux_dspi.c

```diff
@@ -307,6 +307,7 @@ static const struct spi_driver_api spi_mcux_driver_api = {
 			DT_INST_PROP(id, transfer_delay)),		\
 	};								\
 	static struct spi_mcux_data spi_mcux_data_##id = {		\
+		.ctx.owner = NULL,					\
```

Comment:
that's not required, it will be NULL by default.

Reply:
makes life easier